src/mon/Paxos.cc

```diff
@@ -619,6 +619,9 @@ void Paxos::begin(bufferlist& v)
   // and no value, yet.
   assert(new_value.length() == 0);
 
+  //cancle lease
```

Comment:
"cancel", but, the comment doesn't really add anything to the function name, so it seems unnecessary

Reply:
yes, i will delete it